### ISSUE TITLE
Error: http: multiple response.WriteHeader calls

### ISSUE THREAD

**Issue by vpavlin** [fabric8-services/fabric8-jenkins-proxy#128]
```
{"level":"info","msg":"Redirecting to remove token from URL","ns":"vpavlin-jenkins","time":"2018-02-08T11:52:57Z"}
  | 2018/02/08 11:52:57 http: multiple response.WriteHeader calls

```

This prevents token removal from the URL

**Comment by hferentschik** [comment]
> This prevents token removal from the URL

Ahh, good catch.

**Comment by hferentschik** [comment]
Merged via pull request #129
